package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@blackbaud/auth-client",
-  "version": "2.64.1",
+  "version": "2.64.2",
```

Comment:
This is a new feature and not a bug fix, so you should increment the minor version.